py-scripts/real_application_tests/real_browser/lf_interop_real_browser_test.py

```diff
@@ -883,10 +1245,9 @@ def get_stats(self, duration, file_path, iteration_number, resource_list_sorted,
                 os.chdir(self.result_dir)
 
             start_time = datetime.now()
-            while datetime.now() <= end_time or not self.check_gen_cx():
+            while (datetime.now() <= end_time or (not self.check_gen_cx())):
```

Comment:
Nit: Remove redundant parens

Whitespace changes should be submitted as a separate commit